src/commands/functions/create.js

```diff
@@ -145,57 +153,47 @@ const formatRegistryArrayForInquirer = function (lang) {
 
 // pick template from our existing templates
 const pickTemplate = async function () {
-  inquirer.registerPrompt('autocomplete', inquirerAutocompletePrompt)
-  // doesnt scale but will be ok for now
-  const [
-    jsreg,
-    // tsreg, goreg
-  ] = [
-    'js',
-    // 'ts', 'go'
-  ].map(formatRegistryArrayForInquirer)
   const specialCommands = [
-    new inquirer.Separator(`----[Special Commands]----`),
+    new inquirer.Separator(),
```

Comment:
Purely cosmetic change. I personally don't see a reason to draw special attention to these options by using a "Special Commands" header and surrounding the options with `***`.